maxdiffusion_gpu_dependencies.Dockerfile

```diff
@@ -0,0 +1,49 @@
+# syntax=docker/dockerfile:experimental
+ARG BASEIMAGE=ghcr.io/nvidia/jax:base
```

Comment:
@sizhit2 

**Nit:**

We can probaly add a link from where the base image is being dervice or a description even though navigating using the base image link would be sufficient ) i.e. why is this the default
Also probably a Note that this pulls in the latest of the jax:base i.e. built nightly AFAIR and that the tag changes so only the tag ( dated image on that day is stable i.e. does not chage )

**Super Nit:**

If you feel like contributing back to Jax tool box build process you could add a maxdiffusion test since it now depends on jax toolbox base as one of the CI/CD [test](https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/container/test-maxtext.sh).